lib/pages/homepage/donation_card/DonationCard.dart

```diff
@@ -112,9 +109,7 @@ class _DonationCardState extends State<DonationCard> {
                             ),
                           ),
                           Text(
-                            (currentAmount?.toString() ?? "?") +
-                                " / " +
-                                (totalAmount?.toString() ?? "?"),
+                            (currentAmount?.toString() ?? "?") + " / " + (totalAmount?.toString() ?? "?"),
```

Comment:
I have approximately 0 confidence with Dart but it supports interpolation?

Reply:
yes dart supports string interpolation